federatedscope/core/configs/yacs_config.py

```diff
@@ -0,0 +1,600 @@
+# An extended configuration system based on [yacs],
```

Comment:
We need to add these to `LICENSE` too

Reply:
> LGTM, please see the inline comments for minor suggestions. THX!

I made modifications according to the comments